src/views/about/about.scss

```diff
@@ -47,6 +47,10 @@
         padding: 26px 40px 6px;
         width: calc(100% - 80px);
 
+        h2 {
```

Comment:
Just trying to understand this -- is the h2 style used anywhere? Is this just a preemptive change for the future?

Reply:
That styling is for `<h2>`s that I used to replace the `<h3>`s on the page. Some how the changes that I made to the `<h3>`s did not get through. I spent some time to try to figure out why, but I could not find out why those changes did not make it upstream.

I just push those necessary changes, and they are there now.